CONTRIBUTING documentation outdated

**Describe the bug**
Documentation in CONTRIBUTING is outdated/incorrect and does not build the project correctly

**To Reproduce**
```
rm -rf ~/go;
export GOPATH=~/go
export PATH=$PATH:$GOPATH/bin
go get -u github.com/argoproj/argo-cd
OUTPUT: package github.com/argoproj/argo-cd: no Go files in /Users/jopedros/go/src/github.com/argoproj/argo-cd
cd ~/go/src/github.com/argoproj/argo-cd
make dep
OUTPUT:  make: *** No rule to make target `dep'.  Stop. 
# I assumed the new target is dep-ensure
make dep-ensure
OUTPUT:  dep ensure -no-vendor
make
OUTPUT:  find /Users/jopedros/go/src/github.com/argoproj/argo-cd -name debug.test | xargs rm -f
CGO_ENABLED=0 go run vendor/github.com/gobuffalo/packr/packr/main.go build -v -i -ldflags '-X github.com/argoproj/argo-cd/common.version=1.1.2 -X github.com/argoproj/argo-cd/common.buildDate=2019-08-30T11:03:49Z -X github.com/argoproj/argo-cd/common.gitCommit=21bc70be05ae15c15fcd385b97ceefb7d8ac12ac -X github.com/argoproj/argo-cd/common.gitTreeState=clean -extldflags "-static"' -o /Users/jopedros/go/src/github.com/argoproj/argo-cd/dist/argocd ./cmd/argocd
stat vendor/github.com/gobuffalo/packr/packr/main.go: no such file or directory
make: *** [cli] Error 1
```

**Expected behavior**
Build to take place


**Version**
commit 21bc70be05ae15c15fcd385b97ceefb7d8ac12ac (HEAD -> master, origin/master, origin/HEAD)

--- a/Makefile
+++ b/Makefile
@@ -149,6 +149,10 @@ builder-image:
 	docker build  -t $(IMAGE_PREFIX)argo-cd-ci-builder:$(IMAGE_TAG) --target builder .
 	@if [ "$(DOCKER_PUSH)" = "true" ] ; then docker push $(IMAGE_PREFIX)argo-cd-ci-builder:$(IMAGE_TAG) ; fi
 
+.PHONY: dep
+dep:
+	dep ensure -v
+
 .PHONY: dep-ensure
 dep-ensure:
 	dep ensure -no-vendor

